lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -259,73 +259,115 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
+        return enable_default_service_slice_type
+
     return service_mconfig.default_slice_service_type or DEFAULT_NGAP_S_NSSAI_SST
 
 
 def _get_default_slice_differentiator_type_config(service_mconfig: object) -> str:
     """
-    Retrieves default_slice_differentiator config value,If it does not exist
+    Retrieve default_slice_differentiator config value,If it does not exist
     it defaults to 0xffffff.
     Args:
         service_mconfig:
 
     Returns: slice differentiator config value
-    """
+    """  # noqa: DAR101, DAR201
+    enable_default_slice_differentiator_type = get_service_config_value(
+        'mme', 'default_slice_differentiator', None,
+    )
+
+    if enable_default_slice_differentiator_type is not None:
+        return enable_default_slice_differentiator_type
+
     return service_mconfig.default_slice_differentiator or DEFAULT_NGAP_S_NSSAI_SD
 
 
 def _get_amf_name_config(service_mconfig: object) -> str:
     """
-    Retrieves amf_name config value,If it does not exist
+    Retrieve amf_name config value,If it does not exist
     it defaults to Default Name AMF_1.
     Args:
         service_mconfig:
 
     Returns: amf name string
-    """
+    """  # noqa: DAR101, DAR201
+    enable_amf_name_config = get_service_config_value(
+        'mme', 'amf_name', None,
+    )
+
+    if enable_amf_name_config is not None:
+        return enable_amf_name_config
+
     return service_mconfig.amf_name or DEFAULT_NGAP_AMF_NAME
 
 
 def _get_amf_region_id(service_mconfig: object) -> str:
     """
-    Retrieves amf_region_id config value,If it does not exist
+    Retrieve amf_region_id config value,If it does not exist
     it defaults to Default value of 1
     Args:
         service_mconfig:
 
     Returns: amf region id
-    """
+    """  # noqa: DAR101, DAR201
+    enable_amf_region_id = get_service_config_value(
+        'mme', 'amf_region_id', None,
+    )
+
+    if enable_amf_region_id is not None:
+        return enable_amf_region_id
+
     return service_mconfig.amf_region_id or DEFAULT_NGAP_AMF_REGION_ID
 
 
 def _get_amf_set_id(service_mconfig: object) -> str:
     """
-    Retrieves amf_set_id config value,If it does not exist
+    Retrieve amf_set_id config value,If it does not exist
     it defaults to Default String 1
     Args:
         service_mconfig:
 
     Returns: amf set id
-    """
+    """  # noqa: DAR101, DAR201
+    enable_amf_set_id = get_service_config_value(
+        'mme', 'amf_set_id', None,
+    )
+
+    if enable_amf_set_id is not None:
+        return enable_amf_set_id
+
     return service_mconfig.amf_set_id or DEFAULT_NGAP_SET_ID
 
 
 def _get_amf_pointer(service_mconfig: object) -> str:
     """
-    Retrieves amf_pointer config value,If it does not exist
+    Retrieve amf_pointer config value,If it does not exist
     it defaults to Default String 1
     Args:
         service_mconfig:
 
     Returns: amf pointer value
-    """
+    """  # noqa: DAR101, DAR201
```

Comment:
same comment

Reply:
This stops, the doc string error which was coming for no reason.